src/Foundatio.TestHarness/Caching/CacheClientTestsBase.cs

```diff
@@ -626,29 +626,29 @@ public virtual async Task CanGetAndSetDateTimeAsync()
         {
             await cache.RemoveAllAsync();
 
-            DateTime value = SystemClock.UtcNow.Floor(TimeSpan.FromSeconds(1));
+            DateTime value = DateTime.UtcNow.Floor(TimeSpan.FromSeconds(1));
             long unixTimeValue = value.ToUnixTimeSeconds();
             Assert.True(await cache.SetUnixTimeSecondsAsync("test", value));
             Assert.Equal(unixTimeValue, await cache.GetAsync<long>("test", 0));
             var actual = await cache.GetUnixTimeSecondsAsync("test");
             Assert.Equal(value.Ticks, actual.Ticks);
-            Assert.Equal(value.Kind, actual.Kind);
+            //Assert.Equal(value.Kind, actual.Kind);
```

Comment:
Wouldn't we always want to check this as it's round tripping the data, kind could change date math / behavior

Reply:
Kind doesn't exist on offset.